assets/components/productPage/productPageTabs/productPageTabs.jsx

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { classNameWithModifiers } from 'helpers/utilities';
+
+import './productPageTabs.scss';
+
+
+// ---- Types ----- //
+type Tab = {|
+  name: string,
```

Comment:
this is left open to potentially expand in the future with `title` or `href` or anything else that might crop up